### PR TITLE
First round of release, tests are still missing.

### PR DESCRIPTION
**Pull request by jaysonsantos**
This is just a initial release of the open sourced version.
@pedro-teixeira Could you create a travis job for this project? When you do it I will add a travis.yml for it.
I still have to figure out a good way to test it.

### CODE REVIEW

**Comment by pedro-teixeira** [comment]
@jaysonsantos: https://travis-ci.org/eatfirst/Flask-Click-Migrate/

Please let me know when to create an account on PyPi or any other service so we have all using a central account.

**Comment by pedro-teixeira** [comment]
@jaysonsantos what is the status here?

**Comment by jaysonsantos** [comment]
I didn't work on it anymore, probably I will do it next week.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+  - "2.7"
+  - "3.4"
+  - "3.5"
```
Comment (pedro-teixeira) [inline .travis.yml:5]: Why not `3.6`?

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+"""Flask-Click-Migrate."""
+
+from setuptools import setup
+
+setup(
+    name='Flask-Click-Migrate',
+    version='0.0.1',
+    author='Eat First gmbh',
```
Comment (pedro-teixeira) [inline setup.py]: Please change to `EatFirst.com`

**Review by pedro-teixeira — CHANGES_REQUESTED** [review]

**Comment by pedro-teixeira** [comment]
Please fix the `LICENSE` file to have the proper copyright.

File: requirements_test.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+flake8
```
Comment (andreoliwa) [inline requirements_test.txt]: Shouldn't we pin the versions, even for testing?

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    author_email='jayson.reis@eatfirst.com',
+    description='A library to glue flask + alembic + click',
+    url='https://github.com/eatfirst/Flask-Click-Migrate',
+    packages=['flask_click_migrate'],
+    classifiers=[
+        'Development Status :: 4 - Beta',
+        'License :: OSI Approved :: MIT License',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
```
Comment (andreoliwa) [inline setup.py:18]: "3.4" and "3.5", no?
According to the travis.yml file above.

**Review by pedro-teixeira — APPROVED** [review]

**Review by andreoliwa — APPROVED** [review]

**Pull request merged**